examples/hosts/app-integration/external-data/src/model-interface/TaskData.ts

```diff
@@ -6,17 +6,24 @@
 /**
  * Mock model for external task data
  */
-export interface TaskData {
+export interface TaskList {
```

Comment:
Probably use 'I' prefix (`ITaskList`) for consistency here and below.  Also probably want to avoid the name collision with the data object, so maybe this is better to stay as `ITaskData` or `ITaskListData`?